### PR TITLE
Added more links to ignore

### PR DESCRIPTION
**Pull request by lavr001**
#### Description of changes

Ignore some links to unblock migration work

#### Checklist

- [ ] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [ ] PR description included
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] `yarn test` passes and tests are updated/added
- [ ] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by calebpollman — COMMENTED** [review]
Looks like there are a lot of unrelated changes in this PR, can you please clean it up?

File: packages/react/src/primitives/types/tabs.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ export interface BaseTabsProps extends BaseFlexProps {
    * @description
    * The Tabs component only accepts TabItem components as children.
    */
-  children: React.ReactNode;
+  children?: TabItemProps | TabItemProps[];
```
Comment (reesscot) [inline packages/react/src/primitives/types/tabs.ts:16]: What is this change here for?

File: docs/src/pages/[platform]/components/tabs/examples/BasicTabs.tsx [hunk 2]
```diff
@@ -4,5 +4,12 @@ export const BasicTabs = () => (
   <Tabs>
     <TabItem title="Tab 1">Tab 1 Content</TabItem>
     <TabItem title="Tab 2">Tab 2 Content</TabItem>
+    <TabItem title="Tab 3">Tab 3 Content</TabItem>
```
Comment (reesscot) [inline docs/src/pages/[platform]/components/tabs/examples/BasicTabs.tsx:7]: ?

Comment (lavr001) [reply docs/src/pages/[platform]/components/tabs/examples/BasicTabs.tsx]: Not sure how it's got sneaked in, removed

File: packages/react/src/primitives/types/tabs.ts (hunk 1 above)
Comment (lavr001) [reply packages/react/src/primitives/types/tabs.ts]: Removed

**Review by thaddmt — APPROVED** [review]

**Review by reesscot — APPROVED** [review]

File: docs/src/data/ignoredLinks.ts [hunk 3]
```diff
@@ -33,4 +33,8 @@ export const IGNORED_LINKS = [
   'https://www.w3.org/WAI/ARIA/apg/patterns/alert/examples/alert/',
   'https://www.w3.org/WAI/ARIA/apg/patterns/menu-button/',
   'https://www.w3.org/WAI/ARIA/apg/patterns/breadcrumb/',
+  'https://pub.dev/documentation/amplify_authenticator/latest/amplify_authenticator/AuthenticatorStep.html',
+  'https://pub.dev/documentation/amplify_authenticator/latest/amplify_authenticator/AuthenticatorState-class.html',
+  'https://pub.dev/documentation/amplify_authenticator/latest/amplify_authenticator/amplify_authenticator-library.html',
+  'https://docs.flutter.dev/ui/accessibility-and-localization/internationalization',
```
Comment (calebpollman) [inline docs/src/data/ignoredLinks.ts:39]: Non-blocking due to severity, but there should be comments explaining why these are ignored. Future maintainers should have context in to why these were added

**Pull request merged**